mmpose/datasets/dataset_wrappers.py

```diff
@@ -116,7 +87,8 @@ def prepare_data(self, idx: int) -> Any:
         subset_idx, sample_idx = self._get_subset_index(idx)
         # Get data sample from the subset
         data_info = self.datasets[subset_idx].get_data_info(sample_idx)
-        data_info = self.datasets[subset_idx].pipeline(data_info)
+        if hasattr(self.datasets[subset_idx], 'pipeline'):
+            data_info = self.datasets[subset_idx].pipeline(data_info)
```

Comment:
For sub-dataset which does not have the attribute 'pipeline', e.g. `RepeatDataset`, it seems like some necessary conversions such as `KeypointConverter` might be skipped